### ISSUE TITLE
xcodebuild_arguments recieves error in 0.11.0.

### ISSUE THREAD

**Issue by noobs2ninjas** [realm/jazzy#1103]
Hey guys. Thought I'd let you know that moving up to 11.0 broke our jazzy build. Heres the yaml:

```
module: "ParseLiveQuery"
xcodebuild_arguments: ["-scheme", "ParseLiveQuery-iOS"]
theme: "fullwidth"
skip_undocumented: true
clean: true
author_url: "http://parseplatform.org"
github_url: "https://github.com/parse-community/ParseLiveQuery-iOS-OSX"
root_url: "http://parseplatform.org/ParseLiveQuery-iOS-OSX/"
author: "Parse Community"
```

This gave us the following error.

```
/gems/jazzy-0.11.0/lib/jazzy/executable.rb:36:in `execute_command' /.rvm/gems/ruby-2.6.3/gems/jazzy-0.11.0/bin/sourcekitten ["doc", "--spm", "--module-name", "ParseLiveQuery", "--", "-scheme", "ParseLiveQuery-iOS"] (RuntimeError)

Running swift build

Build failed, saved `swift build` log file: /var/folders/lb/0vy6srtd2b7b8yq138w_rfjc0000gn/T/swift-build-96CAAD1B-BC2B-49D6-9A89-094A694C91FE.log
```

The log file says the following `error: unknown option -scheme; use --help to list available options`. Moving down to 10.0 fixed it. Haha unfortunately it took me 2 hours to figure out it wasnt all the build changes I made. Might want to take a look at it as the release files says there where no breaking changes.

**Comment by johnfairh** [comment]
@noobs2ninjas thanks for the report; can you share the file list in the directory you're running `jazzy` from?  It's deciding to use SPM instead of `xcodebuild` which is obviously wrong for your case.

A workaround to keep using 0.11 would be to add `swift_build_tool: xcodebuild` to your config file, but you're right this shouldn't be required -- the intention was to be back-compatible.

**Comment by noobs2ninjas** [comment]
Thanks for the quick reply @johnfairh. If I'm being completely honest I am brand new to Jazzy. Also, I'm super exhausted because I've been pulling my hair out for the last two hours thinking I was to blame for breaking the pull requests everyones been waiting for. I stupidly volunteered to update Jenkins and Circle to Xcode11 with little experience with either. Forgive me but do you think I could have you take a look at it yourself? I work with Parse, the open source database platform and this actually came from our iOS LiveQuery SDK. You can see absolutely everything right here.

https://github.com/parse-community/ParseLiveQuery-iOS-OSX 

I really appreciate it. If thats too much to ask let me know and Ill list everything off. I was just on my way to bed and didnt want to slow you guys down. Thanks!

**Comment by johnfairh** [comment]
I confirm that adding `swift_build_tool: "xcodebuild"` to the `.jazzy.yaml` makes that project's docs build with 0.11.0.

I've fixed the jazzy bug in #1104 - will give it a day or so before a new release for anything else to show up.

**Comment by noobs2ninjas** [comment]
Awesome! Thanks for work around! I’ll keep a lookout for the update!
